parcels/particle.py

```diff
@@ -6,7 +6,7 @@
 from collections import OrderedDict, Iterable
 import matplotlib.pyplot as plt
 import math
-import datetime
+from datetime import timedelta
```

Comment:
Since we import `timedelta` in the tests as `delta`, should we do the same here, for consistency?